corehq/apps/translations/app_translations.py

```diff
@@ -864,8 +864,9 @@ def _update_case_list_translations(sheet, rows, app):
 
         # If it's a graph series configuration item, add it to its parent
         elif row['case_property'].endswith(" (graph series config)"):
-            row['id'] = _remove_description_from_case_property(row)
-            row['series_index'] = row['case_property'].split(" ")[1]
+            trimmed_property = _remove_description_from_case_property(row)
+            row['id'] = trimmed_property.split(" ")[0]
```

Comment:
dont we get case property like `if(xys, asd)` in (graph series config) ?

Reply:
No. The graph config "case_property" values are all automatically generated and of the format `PROPERTY (graph config)` or `PROPERTY SERIES_ID (graph series config)`, and PROPERTY is always an HQ slug like "name" or "x-name".